src/Header/fs-header.css

```diff
@@ -1,5 +1,5 @@
 .fs-header {
-  height: 54.8px;
+  height: 54px;

```

Comment:
On line 10 there is same width, that you fixed. When I write some comment in 90% cases it belongs to all things on your project